compiler/rustc_codegen_llvm/src/gotoc/rvalue.rs

```diff
@@ -788,35 +769,10 @@ impl<'tcx> GotocCtx<'tcx> {
         }
     }
 
-    fn codegen_vtable_methods(
-        &mut self,
-        trait_ref_t: Binder<'tcx, TraitRef<'tcx>>,
-        t: Ty<'tcx>,
-    ) -> Vec<Expr> {
-        //DSN This assumes that we always get the methods back in the same order.
-        let methods = self
-            .tcx
-            .vtable_entries(trait_ref_t)
-            .iter()
-            .cloned()
-            .map(|entry| match entry {
-                ty::VtblEntry::Method(def_id, substs) => {
-                    Some(self.codegen_vtable_method_field(def_id, substs, trait_ref_t, t))
-                }
-                _ => None,
-            })
-            .filter_map(|x| x)
-            .collect();
-        methods
-    }
-
     /// The size and alignment for the vtable is of the underlying type.
     /// When we get the size and align of a ty::Ref, the TyCtxt::layout_of
     /// returns the correct size to match rustc vtable values. Checked via
     /// RMC-compile-time and CBMC assertions in check_vtable_size.
-    ///
-    /// As per tautschn@, the way to get the sizeof an object in gotoc is
-    /// `__CPROVER_POINTER_OFFSET(&((int *)0)[1])`
```

Comment:
Just curious, why was this comment removed?

Reply:
It's a stale comment since `check_vtable_size` below now uses a different Gotoc function, described in the comment there. Thought I would remove since I was touching the size and align code!